src/persistence/EntityPersistExecutor.ts

```diff
@@ -94,9 +94,9 @@ export class EntityPersistExecutor {
 
                     // load database entities for all subjects we have
                     // next step is to load database entities for all operate subjects
-                    // console.time("loading...");
+                    console.time("loading...");
                     await new SubjectDatabaseEntityLoader(queryRunner, subjects).load(this.mode);
```

Comment:
This is where the identity gets 'reloaded' because the identifier is already set. To be fair, even if the identifier wasn't set, the identity shouldn't be reloaded, because it's a new one. It should just go straight to the INSERT.